### PR TITLE
Update sql-quiz.ts

### PR DESCRIPTION
**Pull request by astitva0011**
*About TCL command
*Updated sql-quiz.ts

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -576,6 +576,17 @@ const sqlQuiz = [
       "The ROUND() function takes in two parameters(number and decimals) and rounds the number to the number of decimals specified in the 2nd parameter.",
     Link: "https://dev.mysql.com/doc/refman/5.7/en/precision-math-rounding.html",
   },
+  {
+    Question:
+      "In SQL, What does TCL stand for ",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:581]:
```suggestion
      "In SQL, what does TCL stand for?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 1 above)
Comment (astitva0011) [reply src/data/sql-quiz.ts:581]: > ```suggestion
>       "In SQL, what does TCL stand for?",
> ```

I will do the suggested changes soon

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**